lib/backend/sqlite.cc

```diff
@@ -2,6 +2,7 @@
 
 #include <sqlite3.h>
 #include <fcntl.h>
+#include <cinttypes>
```

Comment:
This is not a C header. If you want a directly backportable patch, it should be <inttypes.h>